GoogleDataTransport/GDTCORLibrary/GDTCORTransformer.m

```diff
@@ -48,8 +48,9 @@ - (instancetype)init {
 
 - (void)transformEvent:(GDTCOREvent *)event
       withTransformers:(NSArray<id<GDTCOREventTransformer>> *)transformers
-            onComplete:(nonnull void (^)(BOOL wasWritten, NSError *error))completion {
+            onComplete:(void (^_Nullable)(BOOL wasWritten, NSError *error))completion {
   GDTCORAssert(event, @"You can't write a nil event");
+  BOOL hadOriginalCompletion = completion != nil;
   if (!completion) {
```

Comment:
if (! hadOriginalCompletion)

Reply:
This would work, but I prefer the semantics of !completion leading to reassigning completion.